yt_dlp/extractor/youtube.py

```diff
@@ -4646,14 +4636,33 @@ def _get_uncropped(url):
                     'preference': -5
                 })
 
+        # Deprecated - remove when old layout is discontinued
         primary_thumbnails = self._extract_thumbnails(
             primary_sidebar_renderer, ('thumbnailRenderer', ('playlistVideoThumbnailRenderer', 'playlistCustomThumbnailRenderer'), 'thumbnail'))
 
+        playlist_thumbnails = self._extract_thumbnails(
+            playlist_header_renderer, ('playlistHeaderBanner', 'heroPlaylistThumbnailRenderer', 'thumbnail'))
+
         if playlist_id is None:
             playlist_id = item_id
 
-        playlist_stats = traverse_obj(primary_sidebar_renderer, 'stats')
-        last_updated_unix = self._parse_time_text(self._get_text(playlist_stats, 2))
+        # Deprecated - remove primary_sidebar_renderer when old layout discontinued
+        # Playlist stats is a text runs array containing [video count, view count, last updated].
+        # last updated or (view count and last updated) may be missing.
+        playlist_stats = get_first(
+            (primary_sidebar_renderer, playlist_header_renderer), (('stats', 'briefStats', 'numVideosText'),))
```

Comment:
I think it would be nice to change `get_first` like this:
```py
def get_first(obj, *keys, **kwargs):
    return traverse_obj(obj, *[(..., *variadic(k)) for k in keys], **kwargs, get_all=False)
```
that'd make this just:
```suggestion
            (primary_sidebar_renderer, playlist_header_renderer), 'stats', 'briefStats', 'numVideosText')
```
similar to how traverse_obj accepts multiple paths

Reply:
hmm I'm not sure, up to you. That'll require changing all other usages and testing them though.

Reply:
why? This change is backward compatible

Reply:
probably better to do this in another commit/pr. we should really merge this